packages/strapi-plugin-content-manager/admin/src/components/Inputs/index.js

```diff
@@ -117,6 +117,8 @@ function Inputs({ autoFocus, keys, layout, name, onBlur }) {
 
   if (type === 'float' || type === 'decimal') {
     step = 'any';
+  } else if (type === 'time' || type === 'datetime') {
+    step = 30;
```

Comment:
So step can be a `number`or a `string`. Is it wanted?

Reply:
Yes it is because the timepicker uses a number to define the step while the input number uses a string as it can be set to `any` [here's the documentation](https://developer.mozilla.org/fr/docs/Web/HTML/Element/Input/number)